README.md

```diff
@@ -10,6 +10,7 @@
 - `RwaInputConduit`: repays DAI.
 - `RwaInputConduit2`: variation of `RwaInputConduit` with an whitelist to control permissions to repay DAI.
 - `RwaToken`: represents the RWA collateral in the system.
+- `RwaTokenFactory`: factory for deploying RwaToken.
```

Comment:
Nitpicking here, but maybe "factory of `RwaToken`s" is a more appropriate way to write this.

```suggestion
- `RwaTokenFactory`: factory of `RwaToken`s.
```